rasusa/2.1.0/Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM ubuntu:jammy as app
+
+ARG RASUSA_VER="2.1.0"
+
+LABEL base.image="ubuntu:jammy"
+LABEL dockerfile.version="1"
+LABEL software="rasusa"
+LABEL software.version="${RASUSA_VER}"
+LABEL description="Randomly subsample sequencing reads to a specified coverage."
+LABEL website="https://github.com/mbhall88/rasusa"
+LABEL license="https://github.com/mbhall88/rasusa/blob/master/LICENSE"
+LABEL maintainer="Jake Garfin"
+LABEL maintainer.email="jake.garfin@state.mn.us"
+LABEL maintainer2="Curtis Kapsak"
+LABEL maintainer2.email="kapsakcj@gmail.com"
+LABEL maintainer3="Erin Young"
+LABEL maintainer3.email="eriny@utah.gov"
+
+# install wget and cleanup apt garbage
+RUN apt-get update && apt-get -y install --no-install-recommends \
+    wget \
+    ca-certificates \
+    procps && \
+    apt-get autoclean && \
+    rm -rf /var/lib/apt/lists/*
+
+# install rasusa
+RUN wget -q https://github.com/mbhall88/rasusa/releases/download/${RASUSA_VER}/rasusa-${RASUSA_VER}-x86_64-unknown-linux-musl.tar.gz && \
+    tar -xvf rasusa-${RASUSA_VER}-x86_64-unknown-linux-musl.tar.gz && \
+    rm -rf rasusa-${RASUSA_VER}-x86_64-unknown-linux-musl.tar.gz && \
+    mkdir /data
+
```

Comment:
It looks like the gnu version has been available since version [0.7.1](https://github.com/mbhall88/rasusa/releases/tag/0.7.1), and I just never noticed. I'd call this a legacy issue that I can resolve.